ink-wrapper-types/Cargo.toml

```diff
@@ -1,27 +1,37 @@
 [package]
 name = "ink-wrapper-types"
 version = "0.6.0"
+description = "Helper library for contract client code generated with ink-wrapper."
+categories = []
+authors = ["Cardinal"]
```

Comment:
```suggestion
authors = ["Cardinal Cryptography"]
```

Reply:
We use `Cardinal` now 🤔 at least that's what I saw...

I'll use both.